source/2020-06-19-the-ember-times-issue-153.md

```diff
@@ -9,19 +9,24 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Read through the RFC to add Type Stability to Ember Addons 💠,
 
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [RFC: Type Stability for Addons 💠](https://github.com/typed-ember/ember-cli-typescript/pull/1158)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) proposed a set of guidelines and tooling recommendations for managing changes as addons adopt [TypeScript](https://www.typescriptlang.org/) throughout the Ember ecosystem, as part of the path to making TypeScript a first-class citizen in Ember as a whole.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In the [RFC](https://github.com/typed-ember/ember-cli-typescript/pull/1158), it proposes to align with [Ember’s SemVer and LTS commitments](https://emberjs.com/releases/) and design workflows to support that policy, so that consumers of Ember addons which publish types are insulated from breaking changes in TypeScript whenever possible. The RFC outlines what needs to happen when we encounter:
+
+- Breaking changes to type definitions
+- Non-breaking changes to type definitions
+- Bug fixes to type definitions
+- Dropping support for previously-supported versions
+
+Read through the full [RFC on Type Stability for Addons[(https://github.com/typed-ember/ember-cli-typescript/pull/1158) and leave some feedback!
```

Comment:
```suggestion
Read through the full [RFC on Type Stability for Addons](https://github.com/typed-ember/ember-cli-typescript/pull/1158) and leave some feedback!
```

Should fix the build, reads great otherwise!

Reply:
ah whoops good catch